src/uu/pgrep/src/pgrep.rs

```diff
@@ -275,6 +289,17 @@ fn process_flag_o_n(
     }
 }
 
+#[cfg(unix)]
+fn parse_signal_value(signal_name: &str) -> UResult<usize> {
+    let optional_signal_value = signal_by_name_or_value(signal_name);
+    match optional_signal_value {
+        Some(x) => Ok(x),
+        None => Err(USimpleError::new(
+            1,
+            format!("Unknown signal {}", signal_name.quote()),
+        )),
+    }
```

Comment:
You can simplify this block by using `ok_or_else`:
```suggestion
    optional_signal_value
        .ok_or_else(|| USimpleError::new(1, format!("Unknown signal {}", signal_name.quote())))
```

Reply:
Indeed, that is better. Changed.